src/components/templates/PartyMap/PartyMap.tsx

```diff
@@ -111,14 +67,9 @@ export const PartyMap: React.FC = () => {
         partygoers={recentVenueUsers}
         selectRoom={selectRoom}
         unselectRoom={unselectRoom}
-        enterSelectedRoom={enterSelectedRoom}
       />
 
-      <RoomModal
-        show={isRoomModalOpen}
-        room={selectedRoom}
-        onHide={unselectRoom}
-      />
+      {selectedRoom && <RoomModal room={selectedRoom} onHide={unselectRoom} />}
```

Comment:
Why was this changed? We explicitly want the `show` prop, and we never want to 'unrender' the `RoomModal` component. This is a regression and should be reverted. (see other comments in this review for further explanation)

Reply:
My initial reasoning was that I wanted to have `room` to be defined there. But I got your point. Will do it in another way

Reply:
Sounds good.